ci/infra/testrunner/testrunner.py

```diff
@@ -171,6 +173,9 @@ def main():
                                help="Use cloud provider integration")
     bootstrap_args.add_argument("-t", "--timeout", type=int, default=300,
                                 help="timeout for waiting a node to become ready (seconds)")
+    bootstrap_args.add_argument("-m", "--registry-mirror", metavar='L',
+                                help="Add to the registry at the first L a mirror at the second L",
+                                nargs=2, action='append')
```

Comment:
Maybe an example of how to use the parameter. I understand it is `--registry-mirror name http://url/to/registry`

Reply:
done